Update sizing cluster article to ping external article (#1889)
giantswarm/docs: src/content/getting-started/sizing-multi-tenancy/index.md

  - How should I separate concerns using several clusters?
  - How many worker nodes should my clusters have?
  - How many control plane nodes should I run?

> last_review_date: 2023-28-08

Reviewer: ```suggestion
last_review_date: 2023-08-28
```

There are a few questions that you might want to answer to determine the right size for your cluster:

1. What kind of environment are you targeting? (short test, development, QA, production)
2. What kind of workloads will be running in there and how many resources do they need?

In general, a cluster that will only be short-lived and for testing purposes can be relatively small and doesn't need much buffer.
Otherwise, you should always account for at least 1 node buffer, so in case one of your nodes goes down or gets rolled due to updates your workloads still have room to be scheduled.

Furthermore, once you want to actually use the cluster productively, we would recommend having at least 4 worker nodes in your cluster.
"Working productively" includes working regularly with any kind of cluster, even if it's officially only a cluster for development or testing.
You don't want your development environment to be down, because of a minor node failure.

> For production usage, the cluster should have at least 5 nodes and a buffer of 2 nodes. For further information on how to size your cluster please [read this learnk8s article](https://learnk8s.io/kubernetes-node-size) which dives into all variables to consider when you don`t know the type and number of nodes to configure for your cluster(s).

Reviewer: ```suggestion
For production usage, the cluster should have at least 5 nodes and a buffer of 2 nodes. For a deep dive on how to size your cluster, please [read this learnk8s.io article](https://learnk8s.io/kubernetes-node-size) which dives into all variables to consider when you don't know the type and number of nodes to configure for your cluster(s).
```

This setup is often repeated for each team, so that each team has 2-5 clusters.

However, with a lot of teams, it might lead to a lot of wasted resources,
 especially if the products of these teams should be working together anyway.
In these cases, it might be better to have a single (or few) production (and a staging) cluster for everyone.
The classification of these clusters should be rather by workloads than by teams.
Thus, you could, for example, have a separate cluster for services that handle

>  sensitive data, and/or one that holds services that need to fulfil certain compliance requirements.

Reviewer: ```suggestion
 sensitive data, and/or one that holds services that need to fulfill certain compliance requirements.
```

(American English, which we mostly prefer)
